models/Member.js

```diff
@@ -0,0 +1,66 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+// add validations later in case of possible errors 
+
+const memberSchema = new mongoose.Schema({
+    first_name: {
+        type: String,
+        required: true,
+    },
+
+    last_name: {
+        type: String,
+        required: true,
+    },
+
+    student_id: {
+        type: String,
+        required: true,
+    },
+
+    department: {
+        type: String,
+        required: true,
+    },
+
+    year_of_education: {
+        type: String,
+        required: true,
+    },
+
+    email: {
+        type: String,
+        required: true
+    },
+
+    first_choice_coord: {
+        type: String,
+        required: true
```

Comment:
first ve second choice için şöyle yapalım: 5 tane koordi seçeneği var ya ve üyenin seçimi de bunlardan biri olmak zorunda. modelde bunu ekleyebiliyoruz. bu nasıl oluyormuş bir bak bulamazsan göstereyim. yey